### ISSUE TITLE
e2e tests won't run

### ISSUE THREAD

**Issue by kraanerk** [AngularClass/angular2-webpack-starter#1113]
- **I'm submitting a ...**
  [x] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
- **What is the current behavior?**
  'npm run e2e' fails with the following reason:
  Error: ⨯ Unable to compile TypeScript
  Conflicting definitions for 'selenium-webdriver' found at 'C:/Users/Keio Kraaner/Downloads/angular2-webpack-starter/node_modules/@types/selenium-webdriver/index.d.ts' and 'C:/Users/Keio Kraaner/Downloads/angular2-webpack-starter/node_modules/protractor/node_modules/@types/selenium-webdriver/index.d.ts'. Consider installing a specific version of this library to resolve the conflict.
- **What is the expected behavior?**
  e2e tests should succeed.
- **Please tell us about your environment:**
  Windows 10
  node v6.8.1
  npm v3.10.8

**Comment by yuanfeiz** [comment]
The typings for protractor are always annoyed, it conflicts with jQuery, with selenium-webdrives.

**Comment by Gideonnn** [comment]
@kraanerk Have you found a workaround that works untill it is fixed?

**Comment by kraanerk** [comment]
@Gideonnn I checked out version 199222047085072d1c6c40364dcfcacb31e07a1d, which does not have this problem.

**Comment by eddedd88** [comment]
I am getting the same error after a fresh clone from the master branch,, could we re-open this?

**Comment by dallastjames** [comment]
+1 still getting this same error on fresh clone of master as well.

**Comment by svieillard** [comment]
Seems to be related with protractor updating to work with TS2.

Got e2e tests running again by updating the following:

`"@types/selenium-webdriver": "2.53.33",`
`"protractor": "4.0.10",`

and removed:

`"@types/protractor": "^1.5.20",`

also don't forget to remove `"protractor",` from tsconfig `compoilerOptions` as well:

```
"types": [
            "hammerjs",
            "jasmine",
            "node",
            "selenium-webdriver",
            "source-map",
            "uglify-js",
            "webpack"
        ]
```

(since protractor comes with their own typings now).

And next add the required imports to *.e2e.ts files like ([ref](https://github.com/angular/protractor/tree/master/exampleTypescript)):

`import { browser, by } from 'protractor';`

**Comment by mdelgadov** [comment]
just a small note: in the import add element, like this:
import {browser, by, element } from 'protractor';

**Comment by sky-coding** [comment]
Thanks @svieillard and @mdelgadov. I put in a PR to fix this on master.

**Comment by glendaviesnz** [comment]
To fix this error with  "protractor": "^4.0.14" I needed to also remove @types/selenium-webdriver as protractor now includes these types also
